README.rst

```diff
@@ -1,3 +1,8 @@
+|travis-badge|_
+
+.. |travis-badge| image:: https://api.travis-ci.org/pypa/warehouse.svg?branch=master
```

Comment:
Please move this definition to the end of the README. It doesn't make sense to have it split but put in the same place.